naurffxiv/src/app/uwu.js

```diff
@@ -1,5 +1,58 @@
 export default function DisplayUWU() {
 	return (
-		<p>Placeholder: Put uwu resources here</p>
+		<>
+		<h1>UWU Strats and Resources</h1>
+			<h2>Table of Contents</h2>
+			UWU Strats by Phases<br></br>
+			General Fight Resources<br></br>
+			UWU Visual Guides and POVs<br></br>
```

Comment:
if these dont need to bolded then we can get rid of the `br` tag. but typically we would wrap the elements inside the `br` tag. @brandon-au is also going to work on the hyperlinks actually showing up in blue so we may not need this at all

Reply:
I think we can remove these for now actually. I would actually like to see a dynamic table of contents on the right side of the page rather than at the top. An example is below.

![image](https://github.com/user-attachments/assets/924e6158-8369-4b23-8168-8cafd1b13e35)

Reply:
Additionally, you can't wrap the <br> tags as you get the error below
![image](https://github.com/user-attachments/assets/25ae3647-c54b-419b-82fa-d5313894347b)